### ISSUE TITLE
Allow Collapsing of Side Panel

### ISSUE THREAD

**Issue by blarosen95** [MicrosoftDocs/feedback#2237]
![Windows Doc Page](https://user-images.githubusercontent.com/26372949/70330437-91520580-180b-11ea-8f4c-2bd200d80889.png)

Some pages on your Documentation (docs.microsoft.com) subdomain can be a bit long. For around 75% of webpages I visit, I like using ctrl+f to find specific things. In the above screenshot, I've circled the pane I'm talking about. This pane takes up some space which I'm not always keen on, however I'm fairly close to indifferent about the screen space it uses. Most of my frustration with it comes up when I'm trying to use a browser's Find feature, but it searches through that side pane too (which can have hundreds and hundreds of matches in it, depending on what I'm looking for).

I'd like to be able to collapse it (probably some JS magic would do the trick). I do know that it will take more than just collapsing the pane to prevent matches from still being found in the pane (at least on Edge and Firefox, haven't checked it with Chrome), as can be seen at https://www.w3schools.com/howto/tryit.asp?filename=tryhow_js_collapse_sidepanel when searching for, say, "about", which still hits 2 matches (one from the code on the left, and one in the collapsed or expanded sidepanel in the WYSIWYG on the right).

I've one idea that might mitigate the fact that browsers' Find feature still matches inside a collapsed sidepanel (when scripted like the example from that previous link): Allowing the user to collapse the contents of the panel as a node. To clarify that, there are already nodes for different parts of the left panel at [https://docs.microsoft.com/en-us/windows/iot-core/develop-your-app/backgroundapplications](https://docs.microsoft.com/en-us/windows/iot-core/develop-your-app/backgroundapplications). If one searches for "unit tests" on that page, it will match the only occurrence which is in the "Develop applications" node. However, if you collapse that node and search again, it doesn't match it anymore.

Perhaps you can put all of those collapse-able nodes inside of another, single collapse-able node at the top of the pane, thus allowing users to collapse that one node and prevent matching both the nodes' contents and the nodes' names? I do realize that this feature then becomes a sort of easter egg by this point, as it would be more of a lifehack when discovered, but it would still allow for a solution.

**Comment by blarosen95** [comment]
Food for thought: If all those nodes are programmatically collapsed when one collapses the sidepanel, it would be less Easter Egg-y and much more intuitive.

**Comment by hbazerbashi** [comment]
I was hoping for the same. Collapsable side panel for Microsoft Documentation. I like to have more space for reading and not having so many lines.

**Comment by spottedmahn** [comment]
yes, I too would like this (I 👍'd the issue, etiquette 😉) but I wanted to make my case:

![image](https://user-images.githubusercontent.com/14079228/142773004-6ff3d853-141f-4906-bc87-ea9a08bc0e75.png)
[source](https://docs.microsoft.com/en-us/azure/container-apps/get-started?tabs=bash)

lots of wasted real estate for "vertical" mode, right?

vs a site like GitHub 😉

![image](https://user-images.githubusercontent.com/14079228/142772978-8ac63d00-b0fa-4974-be95-48f162234549.png)
[source](https://github.com/MicrosoftDocs/feedback/issues)

**Comment by spottedmahn** [comment]
>There's a browser extension that adds a toggle button on top of each document when you browse:
https://www.drewsk.tech/docs-fullview/

[source](https://github.com/MicrosoftDocs/sql-docs/issues/6128#issuecomment-915755391)

---

doesn't seem to be working for me though so mileage might vary 🤷‍♂️
